06.bowling_object/game.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  def initialize(scores)
+    score_index = 0
+    @frames = []
+    (1..10).each do |frame|
+      if frame == 10
+        @frames << Frame.new(frame, scores[score_index], scores[score_index + 1], scores[score_index + 2])
+      elsif scores[score_index] == 'X'
```

Comment:
"X"に特別な意味がある、という知識がGameクラスとShotクラスの両方にあります。
どちらか一方の責務にしたいですね。
たとえば、最初からShotクラスの配列を作っておいて

```suggestion
      elsif shots[score_index].strike?
```

みたいにするとスマートかも？